### PR TITLE
No bid version 1.2.5

### PR DESCRIPTION
**Pull request by redaguermas**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Implemented Video support in the NoBid Prebid Adapter.

- contact email of the adapter’s maintainer: reda.guermas@nobid.io
- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/1930/

### CODE REVIEW

**Comment by redaguermas** [comment]
Can someone help me with the CI/Circle errors. They seem to be related to another package

**Comment by redaguermas** [comment]
After merge from master I am getting an error from this Adapter test "colossussspBidAdapter_spec". Can someone resolve this issue please...

**Comment by redaguermas** [comment]
@harpere , @FilipStamenkovic ,
Can you please review my pull request? It seems that there is a problem with this adapter "colossussspBidAdapter_spec" and it's blocking my pull request. Can you please advise what should we do to resolve this issue?

**Comment by FilipStamenkovic** [comment]
I’ll review this PR first thing tomorrow morning.

**Comment by FilipStamenkovic** [comment]
> "colossussspBidAdapter_spec" and it's blocking my pull request. Can you please advise what should we do to resolve this issue?

I'm not sure what is going on with circleci, nor why tests in `colossussspBidAdapter_spec` are failing, but I see that this has nothing to do with this PR, and tests are passing for your adapter. So, I wouldn't worry about failed tests on circleci.

File: modules/nobidBidAdapter.js [hunk 1]
```diff
@@ -197,10 +196,24 @@ function nobidBuildRequests(bids, bidderRequest) {
...
-    if (siteId && bid.params && bid.params.tags) {
-      newAdunit({div: divid, sizes: sizes, rtb: bid.params.tags, siteId: siteId, placementId: placementId}, adunits);
-    } else if (siteId) {
-      newAdunit({div: divid, sizes: sizes, siteId: siteId, placementId: placementId}, adunits);
+
+    var adType = false;
+    if (bid && bid.mediaTypes && bid.mediaTypes.video && bid.mediaTypes.video.context == 'instream') {
+      adType = 'video';
+    } else {
+      adType = 'banner';
```
[the remark this answers is not in the document]
Comment (redaguermas) [reply modules/nobidBidAdapter.js]: Thanks @FilipStamenkovic , 
We updated the adapter with your comments. The codes 'video' and 'banner' we are using are for our backend and do not match 1 to 1 with BANNER and VIDEO for Prebid. They are independent by design.

**Comment by redaguermas** [comment]
@FilipStamenkovic ,
Can you please review?
Thank you

**Pull request merged**